### ISSUE TITLE
Chrono dependency in Moment module causing conflicts

### ISSUE THREAD

**Issue by ryanhossain9797** [snipsco/rustling-ontology#211]
Cargo.toml in moment module has
`
[dependencies]
chrono = "=0.4.8"
`
It causes conflicts with others using 0.4.11 
such as 

error: failed to select a version for `chrono`.
    ... required by package `rustling-ontology-moment v0.19.3 (https://github.com/snipsco/rustling-ontology?tag=0.19.3#3bb1313d)`
    ... which is depended on by `rustling-ontology v0.19.3 (https://github.com/snipsco/rustling-ontology?tag=0.19.3#3bb1313d)`
    ... which is depended on by `snips-nlu-parsers v0.4.3 (https://github.com/snipsco/snips-nlu-parsers?tag=0.4.3#8cfdae50)`
    ... which is depended on by `snips-nlu-lib v0.65.6 (https://github.com/snipsco/snips-nlu-rs#88a047aa)`
    ... which is depended on by `terminal_alpha_beta v0.1.0 (/home/zireael9797/Storage/WebDev/rust/terminal_alpha_beta)`
versions that meet the requirements `= 0.4.8` are: 0.4.8

all possible versions conflict with previously selected packages.

  previously selected package `chrono v0.4.11`
    ... which is depended on by `discord v0.8.0 (https://github.com/SpaceManiac/discord-rs#767a930b)`
    ... which is depended on by `terminal_alpha_beta v0.1.0 (/home/zireael9797/Storage/WebDev/rust/terminal_alpha_beta)`

failed to select a version for `chrono` which could resolve this conflict

**Comment by mykhailokrainik** [comment]
Hello. Is there any update or any solution to this problem you were experiencing @ryanhossain9797 ?

**Comment by ryanhossain9797** [comment]
> Hello. Is there any update or any solution to this problem you were experiencing @ryanhossain9797 ?

No, I eventually didn't need to deal with this issue any more since I stopped using the crate this has a conflict with.
